newspapers.com.js

```diff
@@ -35,46 +35,58 @@
 	***** END LICENSE BLOCK *****
 */
 
-
-// attr()/text() v2
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
-
-
-function detectWeb(doc, url) {
+function detectWeb(doc) {
```

Comment:
@dstillman what's your view on this? it's easy enough to eslint-escape locally, but this will be a pretty common issue, so might be worth changing globally? On the other hand, the may well be actual cases where `doc` is passed unnecessarily that'd be nice to catch.

Reply:
It'd be fine on a technical level to just remove the unneeded arguments, but our ESLint rules allow unused arguments to be prefixed with an underscore (`function detectWeb(_doc, _url)`), the idea being that it can be helpful to leave in the available parameters in case someone updating the code later wants to make use of them without having to remember or look up the function signature. I'm ambivalent about this, but that's the thinking behind it, at least.

Reply:
I've kept it with @adam3smith 's version for simplicity

Reply:
I'm not sure what you mean, but `detectWeb(doc)` will still trigger a lint error, since `doc` is unused. If we're going to keep the lint rule, this should be `function detectWeb(_doc, _url)`.